spec/presenters/hyrax/collection_presenter_spec.rb

```diff
@@ -3,15 +3,26 @@
   subject(:presenter) { described_class.new(solr_doc, ability) }
   let(:collection) do
     build(:hyrax_collection,
+          id: 'adc12v',
+          title: ['A clever title'])
+  end
+  let(:active_fedora_collection) do
+    build(:collection_lw,
```

Comment:
With basic metadata removed from PcdmCollection, tests in the presenter that try to access basic metadata by delegating to the solr_doc fail.  Basic metadata is not required for collections, so it is unclear when these methods would be called.  Perhaps a refactor could be explored to see if there is another pattern for supporting additional metadata methods.

For now, use an ActiveFedora collection to maintain the tests of metadata methods.  A follow-on PR will add a generated collection resource class that will include basic metadata for testing.